packages/fxa-content-server/server/lib/amplitude.js

```diff
@@ -400,6 +401,46 @@ function receiveEvent(event, request, data) {
     return;
   }
 
+  const rawEvent = {
+    event,
+    context: {
+      eventSource: 'content',
+      version: VERSION,
+      emailTypes: EMAIL_TYPES,
+      userAgent: request.headers['user-agent'],
+      ..._.pick(data, [
+        'deviceId',
+        'devices',
+        'emailDomain',
+        'emailSender',
+        'emailService',
+        'entrypoint_experiment',
+        'entrypoint_variation',
+        'entrypoint',
+        'experiments',
+        'flowBeginTime',
+        'flowId',
+        'lang',
+        'location',
+        'marketingOptIn',
+        'newsletters',
+        'planId',
+        'productId',
+        'service',
+        'syncEngines',
+        'templateVersion',
+        'uid',
+        'userPreferences',
+        'utm_campaign',
+        'utm_content',
+        'utm_medium',
+        'utm_source',
+        'utm_term',
+      ]),
+    },
+  };
+  logger.info('rawAmplitudeData', rawEvent);
```

Comment:
Where's the config flag to turn this on?

Reply:
D'oh!  I'll add it.  Thanks!

Reply:
Config flag added.

Reply:
...but has incorrect default :'(  sorry

Reply:
Updated the default to false.